lib/classes/PluginManager.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+
+const Utils = require('./Utils');
+const path = require('path');
+const has = require('lodash').has;
+const forEach = require('lodash').forEach;
```

Comment:
is this require necessary? I see that you've been using the built in `[].forEach()` method ... which is probably sufficient.